### PR TITLE
Issue08-Populate grid via URL

### PR DESCRIPTION
**Pull request by ArCiGoUnoSquare**
* Waiting for approval

### CODE REVIEW

**Comment by geoperez** [comment]
And the conflicts?

File: test/Grid.spec.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+import { assert, expect } from 'chai';
 import Grid from '../src/Grid/Grid';
 import Paper from 'material-ui/Paper';
 import PropTypes from 'prop-types';
 import React from 'react';
-import sinon from 'sinon';
+import sinon, { spy } from 'sinon';
 import Enzyme, { mount, shallow } from 'enzyme';
 import Table, { TableBody, TableCell, TableHead, TableRow } from 'material-ui/Table';
+import Axios from 'axios';
```
Comment (geoperez) [inline test/Grid.spec.js]: Why do you need this? If you need use Axios use https://www.npmjs.com/package/axios-mock-adapter

Comment (ArCiGoUnoSquare) [reply test/Grid.spec.js]: I'm going to remove the unused import

File: src/Grid/Grid.js [hunk 2]
```diff
@@ -24,13 +25,20 @@ class Grid extends React.Component{
     page: this.props.page,
     rowsPerPage: this.props.rowsPerPage,
     data: this.props.data,
-    columns: this.props.columns
+    serverUrl: this.props.serverUrl,
+    records: []
```
Comment (kadosh) [inline src/Grid/Grid.js]: So, now we have data and records? Why not only one?

**Comment by geoperez** [comment]
Travis is failing....

File: src/Grid/Grid.js (hunk 2 above)
Comment (ArCiGoUnoSquare) [reply src/Grid/Grid.js]: I removed it, but for future PR (not of my own): By this time there is an other branch that uses the variable `data` only for testing purposes. The other branch doesn't have the capability to populate the `<Grid />` component via URL

**Comment by kadosh** [comment]
Error handling when loading the data is missing. Please add it, at least a placeholder with a Console.log

**Pull request merged**